semialign/src/Data/Semialign/Internal.hs

```diff
@@ -139,16 +139,6 @@ oops = error . ("Data.Align: internal error: " ++)
 --          ≡ mapMaybe justHere (toList (align x y))
 -- @
 --
---
--- And an addition property if @f@ is 'Foldable',
```

Comment:
why this is removed?

Reply:
Both the sentence and the law are duplicated above.